src/BalanceClient.ts

```diff
@@ -0,0 +1,22 @@
+const {providers} = require("near-api-js");
+
+export class BalanceClient {
+
+  static async balanceOfOnNear(contractId: string, accountId: string): Promise<number> {
```

Comment:
name this method `ftBalanceOfOnNear` as it can be confusing since address can also hold native near